src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -151,10 +154,26 @@ protected function setMappingDriverConfig(array $mappingConfig, string $mappingN
      * Returns false when autodetection failed, an array of the completed information otherwise.
      *
      * @return array|false
+     *
+     * @deprecated since Symfony 5.4, use "getBundleMappingDriverDefaultConfig()" instead
      */
     protected function getMappingDriverBundleConfigDefaults(array $bundleConfig, \ReflectionClass $bundle, ContainerBuilder $container)
     {
-        $bundleDir = \dirname($bundle->getFileName());
+        trigger_deprecation('symfony/doctrine-bridge', '5.4', 'Method "%s()" is deprecated, use "getBundleMappingDriverDefaultConfig()" instead.', __METHOD__);
+
+        return $this->getBundleMappingDriverDefaultConfig($bundleConfig, ['path' => \dirname($bundle->getFileName()), 'namespace' => $bundle->getNamespaceName()], $container);
```

Comment:
Now that I think of it, instead of manually building the `$bundleMetadata` info, it could be fetched from the container builder?